src/socks/__init__.py

```diff
@@ -1,4 +1,9 @@
-"""SocksiPy - Python SOCKS module.
+# pylint: disable=too-many-arguments,global-statement,too-many-branches
```

Comment:
This file will be removed once asyncore Socks resolver is finished, so I don't care about it. For P2P connections over Socks we're already using my port of this to asyncore.